docs/Variables.md

```diff
@@ -0,0 +1,729 @@
+  - Note that this is passed to the compiler and can thus be used in your code
+
+- THEOS_STAGING_DIR (str)
+  - File path for directory to house stage bits and pieces (default: .theos/_/)
+
+- THEOS_STAGING_DIR_NAME (str)
+  - Name for the staging directory (default: _)
+
+- THEOS_LAYOUT_DIR (str)
+  - File path for directory to house items to move onto the target's filesystem during install (default: $THEOS_PROJECT_DIR/layout/)
+  - The containing directory structure is used as an install guide
+    - Exception to this is $THEOS_LAYOUT_DIR/DEBIAN/ which is used to install Debian packaging related files
+      - Such files include a [control](https://www.debian.org/doc/debian-policy/ch-controlfields.html) file, which will get added to the `status` file (i.e., /var/lib/dpkg/status) on-device, and [maintainer scripts](https://wiki.debian.org/MaintainerScripts), which, after being run, will be placed in /var/lib/dpkg/info/.
+
+- THEOS_LAYOUT_DIR_NAME (str)
+  - Name for the layout directory (default: layout)
+
+- THEOS_BUILD_DIR (str)
+  - File path for directory to house bits and pieces relevant to building (default: ./)
+
+- THEOS_OBJ_DIR (str)
+  - File path for directory to house object files (default: .theos/obj/)
+
+- THEOS_OBJ_DIR_NAME (str)
+  - Name for the object directory (default: obj/$THEOS_CURRENT_ARCH)
+
+- THEOS_PACKAGE_DIR (str)
+  - File path for directory to house built packages (default: $THEOS_BUILD_DIR/packages/)
+
+- THEOS_PACKAGE_DIR_NAME (str)
+  - Name for the packages directory (default: packages)
+
+- THEOS_PACKAGE_NAME (str)
+  - Name of the current project as defined by `Package:` in the `control` file
+
+- THEOS_PACKAGE_ARCH (str)
+  - Architecture of the current project as defined by `Architecture:` in the `control` file
+
+- THEOS_PACKAGE_BASE_VERSION (num)
+  - Version of the current project as defined by `Version:` in the `control` file
+
+- THEOS_DEVICE_USER (str)
+  - User to use when installing to target device (default: root)
+
+- THEOS_DEVICE_IP (num)
+  - IP to use when installing to target device
+
+- THEOS_DEVICE_PORT (num)
+  - Port to use when installing to target device
+    - Left up to ssh to determine default or user to provide
+
+- THEOS_SUDO_COMMAND (str)
+  - Target `sudo` binary (default: `sudo`)
+
+- THEOS_CURRENT_ARCH (str)
+  - Current architecture being targeted in build process
+
+- THEOS_TARGET_NAME (str)
+  - Lowercase name of the target platform
+
+- THEOS_SUBPROJECT_PRODUCT (str) ----- **internal ?**
+
+- THEOS_SCHEMA (str)
+  - Schema to build for "Release," "Debug," or "" (default: Debug)
+    - This will adjust what variables are enabled/disabled in Theos' internal configuration
+
+- THEOS_PLATFORM_NAME (str)
+  - Lowercase name of the current host platform (default: `$(shell uname)`)
+
+- THEOS_RSYNC_EXCLUDES (str)
+  - Files/file extensions to exclude in the `rsync` copy commands used throughout the project (default: _MTN .git .svn .DS_Store ._*)
```

Comment:
Can you wrap the default value in an inline code block `_MTN .git .svn .DS_Store ._*` or escape the underscores "\\\_MTN .git .svn .DS_Store .\\\_*"?
Otherwise it shows as:
> _MTN .git .svn .DS_Store ._*

Reply:
Maybe all default values can be in code blocks? Some already do so it would make it consistent and prevent formatting issues